mountpoint-s3/src/cli.rs

```diff
@@ -372,10 +392,18 @@ impl CliArgs {
             filter
         };
 
+        let _event_log_directory: Option<PathBuf> = None;
+        #[cfg(feature = "event_log")]
+        let _event_log_directory = if _is_fork_parent {
```

Comment:
Ideally parent needs it's own log file then (otherwise there is a possibility of mixed up log records). Today parent and child will log to the same file (but I think there is [a possibility](https://github.com/awslabs/mountpoint-s3/blob/main/mountpoint-s3/src/logging.rs#L97) of them writing to different files, so it's kind of an undefined behaviour). I don't want to create a separate file for parent now, neither we can allow a UB in case of the event log, so I decided to remove event logging from parent.

LMK if you have other thoughts though, otherwise I can just add a comment with reasoning.

NB. all of the mount errors [must be in child](https://github.com/awslabs/mountpoint-s3/blob/main/mountpoint-s3/src/cli.rs#L466), I don't expect any to come from the parent

Reply:
I suspect there are failures in the parent (e.g. failed to instantiate/talk to child) that we may want to log eventually. And we may need to fix the UB.

But okay to leave it out of this PR. Just comment on it.